community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/planner/logical/plans/Sargable.scala

```diff
@@ -57,6 +57,16 @@ object AsPropertySeekable {
   }
 }
 
+//This is when dynamic properties are used
+object AsDynamicPropertyNonSeekable {
```

Comment:
It is a little confusing to have non-sargable stuff living in Sargable, but maybe it is just me.